$NetBSD: patch-ac,v 1.3 2010/12/25 11:35:06 obache Exp $

* portability fix
* DragonFly also not require extra libs for OpenSSL.

--- configure.in.orig	2010-05-25 21:52:46.000000000 +0000
+++ configure.in
@@ -37,7 +37,7 @@ LT_INIT
 dnl
 dnl Find programs
 dnl
-if test "z$RM" == "z" ; then 
+if test "z$RM" = "z" ; then 
     AC_PATH_PROG(RM, rm, /bin/rm)
 fi
 AC_PATH_PROG(CP, cp, /bin/cp)
@@ -127,7 +127,7 @@ AM_CONDITIONAL(SHAREDLIB_HACK, [test "z$
 dnl ==========================================================================
 dnl Hack for autoconf version mismatch
 dnl ==========================================================================
-if test "z$shrext" == "z" ; then 
+if test "z$shrext" = "z" ; then 
     shrext=$shrext_cmds
 fi
 
@@ -348,6 +348,7 @@ elif test "z$with_openssl" != "z" ; then
 	*-*-osf5*) openssl_exlibs=;;
 	*-*-openbsd*) openssl_exlibs=;;
 	*-*-netbsd*) openssl_exlibs=;;
+	*-*-dragonfly*) openssl_exlibs=;;
 	#FIXME: check if lib "dl" is required
 	*)          openssl_exlibs=-ldl;;
     esac
@@ -505,7 +506,7 @@ else
     XMLSEC_CRYPTO_DISABLED_LIST="$XMLSEC_CRYPTO_DISABLED_LIST openssl"
 fi
 
-AM_CONDITIONAL(XMLSEC_NO_OPENSSL, test "z$XMLSEC_NO_OPENSSL" == "z1")
+AM_CONDITIONAL(XMLSEC_NO_OPENSSL, test "z$XMLSEC_NO_OPENSSL" = "z1")
 AC_SUBST(XMLSEC_NO_OPENSSL)
 AC_SUBST(OPENSSL_CFLAGS)
 AC_SUBST(OPENSSL_LIBS)
